OpenSDS_Volume_Actions_Design.md

```diff
@@ -0,0 +1,92 @@
+# Title
+
+Status: Pending
+
+Version: Beta
+
+Author(s): PengYiOfOpenSDS
+
+## Summary
+
+Extends the size of a volume. 
+
+## Motivation
+
+Extends the size of a volume to a requested size, in gibibytes (GiB). 
```

Comment:
I think the motivation section needs to state a little more about why we want this. So why is the ability to extend a volume desired and what is it needed for.